src/include/serial_inline.h

```diff
@@ -178,7 +178,7 @@ __wt_col_append_serial(WT_SESSION_IMPL *session, WT_PAGE *page, WT_INSERT_HEAD *
      * we added cannot be discarded while visible to any running transaction, and we're a running
      * transaction, which means there can be no corresponding delete until we complete.
      */
-    __wt_cache_page_inmem_incr(session, page, new_ins_size);
+    __wt_cache_page_inmem_incr(session, page, new_ins_size, true);
```

Comment:
Just to add a note for the next reviewers, it looks like we only set to `true` the new arg `new_update` in this function.

Reply:
There are 3: `__wt_update_serial`, `__wt_insert_serial`, `__wt_col_append_serial` - the ones called from `__wt_row_modify` and `__wt_col_modify`.

Reply:
Oops, sorry I meant "in this file"